lua/starfall/libs_sv/ply.lua

```diff
@@ -20,14 +20,17 @@ local function getPly( this )
     SF.Throw( "Player is not valid.", 3 )
 end
 
-local function canUsePlyCore( ply )
-    return ply:IsAdmin()
+local function checkPlyCorePerms( ply )
+    local bool = ply:IsAdmin() -- Did it like this in the case more ranks need to get added
+    if bool then
+        SF.Throw( "You must be an admin to use this function!", 2 
+    end
```

Comment:
Just a suggestion. Might be a more extensible way of doing it.

```suggestion
local function noAccess( message )
    return SF.Throw( message or "You don't have access to that function!" )
end
    
local function checkPlyCorePerms( ply )
    if not ply:IsAdmin() then return noAccess() end
```